src/worker/core/trade/summary.ts

```diff
@@ -2,6 +2,8 @@ import { team } from "..";
 import { idb } from "../../db";
 import { g, helpers } from "../../util";
 import type { TradeSummary, TradeTeams } from "../../../common/types";
+import { league } from "src/worker/core";
+league.loadGameAttributes();
```

Comment:
Delete these lines, they are not necessary and will cause errors in some situations.